src/Build.UnitTests/Graph/GetCompatiblePlatformGraph_Tests.cs

```diff
@@ -22,6 +23,9 @@
 
 #nullable disable
 
+// The static graph mirrors the negotiation during build to determine plartform for each node.
+// These tests mirror GetCompatiblePlatform_Tests.cs in order to make sure they both are in sync.
```

Comment:
Can this be a summary of the class itself?

Reply:
Yes. added it into summary